CMakeLists.txt

```diff
@@ -309,6 +309,14 @@ cmake_dependent_option(ENABLE_PERL "if enabled, the perl swig bindings will be b
 set(WANT_BASE64 TRUE CACHE BOOL "wants builtin Base64")
 set(WANT_XBASE64 FALSE CACHE BOOL "wants builtin XBase64")
 
+# ---------- Documentation
+
+# Due to the time it takes to build documentation on every change,
+# we choose to disable documentation by default. Only when ENABLE_DOCS==TRUE
+# will docs be built and added to the `make install` target.
+
+option(ENABLE_DOCS "enables documentation building -- suggests doxygen, asciidoc" FALSE)
```

Comment:
I can also change the option to `DISABLE_DOCS` and enable it by default. Thoughts?

Reply:
I think ENABLE_DOCS is better, because it's consistent with ENABLE_TESTS, ENABLE_PERL, etc.

I think DISABLE_DOCS would cause a small confusion with the double negation if you want to enable the documentation.